websites/G/GenoAnime/presence.ts

```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({
+  clientId: "810203651317432351"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "geno",
+    details: "Genoanime !!!"
+  },
+title = document.title; //title of the page
+
+  if (document.location.pathname == "/" ) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Exploring Genoanime";
+  } else if (document.location.pathname.includes("/browse")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Exploring Genoanime library";
+    presenceData.buttons = [{label:"View Library",url: document.location.href}];
+  }
+  else if (document.location.pathname.includes("/details")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Browsing on Genoanime";
+  } 
```

Comment:
```suggestion
    presenceData.details = "Browsing...";
  } 
```
Should be enough since it already says "Playing Genoanime"